extras/bindings/fortran/include/precice/SolverInterfaceFortran.hpp

```diff
@@ -546,10 +502,10 @@ void precicef_set_quad_(
  * IN:  meshID, firstVertexID, secondVertexID, thirdVertexID, fourthVertexID
  * OUT: -
  *
- * @copydoc precice::SolverInterface::setMeshQuadWithEdges()
+ * @copydoc precice::SolverInterface::setMeshQuad()
  *
  */
-void precicef_set_quad_we_(
+void precicef_set_quad_(
```

Comment:
Seems like the bulk methods are still missing in the header of the Fortran bindings?